src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -46,6 +48,10 @@ function getTypeOfVisit(id) {
 export function transformVAOSAppointment(appt) {
   const isCC = appt.kind === 'cc';
   const isVideo = appt.kind === 'telehealth';
+  const vistaId = appt.locationId?.substr(0, 3);
+  const timezone = getTimezoneBySystemId(vistaId)?.timezone;
+
+  const start = timezone ? moment(appt.start).tz(timezone) : moment(appt.start);
```

Comment:
@jenniemc's PR has similar changes, so I'll remove these changes if that gets merged first.